replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/SyncHelper.java

```diff
@@ -236,6 +228,32 @@ private Filter buildFilter() {
     return finalFilter;
   }
 
+  @Nullable
+  private Date getModifiedAfter() {
+    final ReplicationStatus replicationStatus =
+        history
+            .getReplicationEvents(config.getName())
+            .stream()
+            .filter(s -> s.getStatus().equals(Status.SUCCESS))
+            .findFirst()
+            .orElse(null);
+
+    if (replicationStatus == null) {
+      return null;
+    }
+
+    if (replicationStatus.getLastMetadataModified() != null) {
```

Comment:
✏️ How about moving this logic inside the `ReplicationStatus` class and let it report the right date?